test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -188,6 +188,29 @@ test('HTMLElement.style', function () {
     }
 });
 
+test('image (GH-1502)', function () {
+    var imageNS       = document.createElementNS('http://www.w3.org/2000/svg', 'image');
+    var nativeImageNS = nativeMethods.createElementNS.call(document, 'http://www.w3.org/2000/svg', 'image');
+    var url           = 'https://google.com:1888/index.html?value#yo';
+    var proxyUrl      = urlUtils.getProxyUrl(url);
+
+    imageNS.href.baseVal = url;
+    nativeMethods.baseValSetter.call(nativeMethods.imageHrefGetter.call(nativeImageNS), url);
+
+    strictEqual(nativeMethods.getAttribute.call(imageNS, 'href-hammerhead-stored-value'), url);
```

Comment:
Add to top
`var DomProcessor      = hammerhead.get('../processing/dom');`
and use
`DomProcessor.getStoredAttrName('href')`